lambdas/bearer-token-handler/src/totp-code-generator.ts

```diff
@@ -0,0 +1,35 @@
+import { generateConfig, Totp } from "time2fa";
+import { Logger } from "@aws-lambda-powertools/logger";
+
+const TOTP_TTL_IN_SECONDS = 30;
+const TOTP_HASH = "sha512";
+const TOTP_LENGTH = 8;
+
+const logger = new Logger();
+
+export async function generateTotpCode(
+  SecretString: string
+): Promise<{ totp: string }> {
+  try {
+    if (!SecretString) {
+      throw new Error("No secret string present.");
+    }
+    const totp_config = generateConfig({
```

Comment:
I know this was done a while ago, we can improve name totp_config to `totpConfig` , totp_code to  `totpCode`